server/src/locker.js

```diff
@@ -0,0 +1,52 @@
+const csvFilePath = "./BikeRackData.csv";
+const csvToJSON = require("csvtojson");
+const NodeGeocoder = require("node-geocoder");
+
+// have csv json, use new library, get 1 adress from csv,
+//use new lib to conver to lat long, once u get, put in array of object, export that new funtion
+
+let options = {
+  provider: "google",
+  httpAdapter: "https",
+  apiKey: "AIzaSyAzi7L69BENzi4MiAH4751HBwOPWtAJS0Q",
```

Comment:
Hide your API key. Use dotenv.

Reply:
Squash this commit, the repo is public so this key will be visible to everyone.

Reply:
Added .env,
changed api key on google, so this key no longer exists.